--- dbview-1.0.3.orig/Makefile	Thu Sep 26 22:05:30 1996
+++ dbview-1.0.3/Makefile	Wed Jul  4 22:04:37 2001
@@ -19,7 +19,7 @@
 MAN=man
 
 CC= gcc
-CFLAGS= -O6 -fomit-frame-pointer -Wall -pipe
+CFLAGS= $(OPT) -Wall -pipe
 # CFLAGS= -g -fomit-frame-pointer -Wall -pipe
 
 # Look where your install program is
@@ -43,10 +43,10 @@
 	makedepend *.c
 
 install: all
-	$(INSTALL) -d -o root -g root -m 755 $(bindir)
-	$(INSTALL) -s -o root -g root -m 755 dbview $(bindir)
-	$(INSTALL) -d -o $(MAN) -g $(MAN) -m 755 $(mandir)/man1
-	$(INSTALL) -o $(MAN) -g $(MAN) -m 644 dbview.1 $(mandir)/man1
+	$(INSTALL) -d -m 755 $(DESTDIR)$(bindir)
+	$(INSTALL) -m 755 dbview $(DESTDIR)$(bindir)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -m 644 dbview.1 $(DESTDIR)$(mandir)/man1
 
 clean:
 	rm -f dbview *.o *.log *~ *.orig Makefile.bak;
