--- dbview-1.0.3.orig/Makefile	Thu Sep 26 22:05:30 1996
+++ dbview-1.0.3/Makefile	Wed Jul  4 22:04:37 2001
@@ -19,7 +19,7 @@
 MAN=man
 
 CC= gcc
-CFLAGS= -O6 -fomit-frame-pointer -Wall -pipe
+CFLAGS= $(OPT) -Wall -pipe
 # CFLAGS= -g -fomit-frame-pointer -Wall -pipe
 
 # Look where your install program is
@@ -27,7 +27,7 @@
 INSTALL = /usr/bin/install
 prefix = /usr
 bindir = $(prefix)/bin
-mandir = $(prefix)/$(MAN)
+mandir = $(prefix)/share/$(MAN)
 
 obj = db_dump.o dbview.o version.o
 
@@ -43,10 +43,10 @@
 	makedepend *.c
 
 install: all
-	$(INSTALL) -d -o root -g root -m 755 $(bindir)
-	$(INSTALL) -s -o root -g root -m 755 dbview $(bindir)
-	$(INSTALL) -d -o $(MAN) -g $(MAN) -m 755 $(mandir)/man1
-	$(INSTALL) -o $(MAN) -g $(MAN) -m 644 dbview.1 $(mandir)/man1
+	$(INSTALL) -d -m 755 $(DESTDIR)$(bindir)
+	$(INSTALL) -m 755 dbview $(DESTDIR)$(bindir)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -m 644 dbview.1 $(DESTDIR)$(mandir)/man1
 
 clean:
 	rm -f dbview *.o *.log *~ *.orig Makefile.bak;
@@ -56,13 +56,5 @@
 
 # DO NOT DELETE
 
-db_dump.o: db_dump.h /usr/include/fcntl.h /usr/include/features.h
-db_dump.o: /usr/include/sys/cdefs.h /usr/include/sys/types.h
-db_dump.o: /usr/include/linux/types.h /usr/include/asm/types.h
-db_dump.o: /usr/include/gnu/types.h /usr/include/linux/fcntl.h
-db_dump.o: /usr/include/sys/stat.h /usr/include/linux/stat.h
-db_dump.o: /usr/include/stdio.h /usr/include/libio.h /usr/include/_G_config.h
-db_dump.o: /usr/include/unistd.h /usr/include/posix_opt.h
-db_dump.o: /usr/include/confname.h /usr/include/malloc.h
-dbview.o: version.h db_dump.h /usr/include/stdio.h /usr/include/libio.h
-dbview.o: /usr/include/_G_config.h /usr/include/getopt.h
+db_dump.o: db_dump.h
+dbview.o: version.h db_dump.h
